geometry_msgs/PoseStamped required_pose
---
float64 distance
